subsys/fs/ext2/Kconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 Antmicro
+# SPDX-License-Identifier: Apache-2.0
+
+# Ext2 file system
+
+config FILE_SYSTEM_EXT2
+	bool "Ext2 file system support"
+	depends on FILE_SYSTEM
+	help
+	  Enable Ext2 file system support.
+
+module = EXT2
+module-str = Ext2
+source "subsys/logging/Kconfig.template.log_config"
+
+if FILE_SYSTEM_EXT2
+
+menu "Ext2 file system Settings"
+	visible if FILE_SYSTEM_EXT2
+
+config EXT2_HEAP_SIZE
+	int "Heap size"
+	default 10240
+	help
+	  Memory for allocation of internal structures (e.g. inodes).
```

Comment:
I may be missing something, but I don't see this Kconfig option being used anywhere (anymore?).

Reply:
Removed.